tests/libvmi/network.go

```diff
@@ -62,7 +61,7 @@ func WithPasstInterfaceWithPort() Option {
 // InterfaceDeviceWithMasqueradeBinding returns an Interface named "default" with masquerade binding.
 func InterfaceDeviceWithMasqueradeBinding(ports ...kvirtv1.Port) kvirtv1.Interface {
 	return kvirtv1.Interface{
-		Name: DefaultInterfaceName,
+		Name: kvirtv1.DefaultPodNetwork().Name,
```

Comment:
One benefit of keeping this "duplicated" constant is that the test would fail (?) if you for whatever reason change the one defined by `v1`

Reply:
@xpivarc I don't think we ever check that any other string in `kubevirt.io/api/core/v1/types.go` stays constant across PRs, we just use the api-defined variables and constants. Is there anything special about this `default` string to merit a test?

Do you see other benefits to exposing `DefaultInterfaceName`?

Note that I don't see a current or imminent abuse of this constant; the motivation for this PR is just an attempt to simplify the API of `libvmi` and to avoid public constants.